client/my-sites/earn/memberships/index.jsx

```diff
@@ -440,7 +440,7 @@ class MembershipsSection extends Component {
 						) }
 					>
 						<NoticeAction href={ `/earn/payments-plans/${ this.props.siteSlug }` } icon="create">
-							{ this.props.translate( 'Add a Payment Plan' ) }
+							{ this.props.translate( 'Add a payment plan' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.